### PR TITLE
Fix missing backtick in the description, move sample code in a blockq…

### PR DESCRIPTION
**Pull request by szib**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

Closing backtick was missing in the sample code. Moved the code and result into blockquotes.

### CODE REVIEW

**Comment by raisedadead** [comment]
Thanks @szib, Just a query any open issues about this to be tagged?, its alright if there are none.

Pro tip: Keep commit messages shorter ( <=50 Chars )
https://chris.beams.io/posts/git-commit/

**Comment by szib** [comment]
#13704 relates to it, but it does not fix that issue. OK. I'll keep them shorter. Thanks.

**Review by systimotic — CHANGES_REQUESTED** [review]
Nice work @szib, thank you for contributing! Could you bring back the `div` tags? You will have to escape the `<` and `>` characters as `&lt;` and `&gt;`

**Comment by systimotic** [comment]
I just noticed, escaping `<li>` in line 554 as `&lt;li&gt;` should fix the issue. While you're at it, could you also wrap `${variable}` in the description in `<code>` tags? Those don't need to be escaped.

**Comment by szib** [comment]
@systimotic I missed your last comment and updated the PR without wrapping ${variable}. I'll update it again soon. Sorry for making mistakes, I'm new to contributing. I did not think that errors in the description were the cause of #13704, but I believe it is fixed now. I wanted to fix only the missing backtick in the sample code, that's why I did not refer to that issue.

**Comment by szib** [comment]
"If you have trouble finding backticks, it's the ` key, to the left of your 1; and has ¬ on it."
I changed ~ to ¬ as the backtick key on my UK keyboard has ¬ on it, not ~. I thought it was a mistake, but I've just realised UK and US keyboards are different. Probably this sentence need to be rephrased, but no idea how.

**Comment by systimotic** [comment]
@szib Yeah, it's different on all keyboard layouts. Perhaps we should link [this superuser answer](http://superuser.com/a/254077) instead?

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 1]
```diff
@@ -542,24 +542,17 @@
...
         "A lot of thing happened in there.",
-        "First, the ${variable} syntax. It's a template literal. Basically, you won't have to use concatenation with + anymore. Just drop the variable in your string, wrapped with ${ and }.",
+        "First, the <code>${variable}</code> syntax. It's a template literal. Basically, you won't have to use concatenation with + anymore. Just drop the variable in your string, wrapped with ${ and }.",
         "Second, we used backticks, not quotes, to wrap around the string. Notice that the string is multi-line.",
         "In ES6, back-ticks give you more robust string creation ability.",
         "Instructions",
         "Use proper template literal syntax with backticks.",
-        "The expected output is each entry of result object's failure array, wrapped inside an <li> element, with class attribute text-warning.",
-        "If you have trouble finding backticks, it's the ` key, to the left of your 1; and has ~ on it."
+        "The expected output is each entry of result object's failure array, wrapped inside an element, with class attribute text-warning.",
```
Comment (systimotic) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: You removed the `<li>` here. Instead, please escape it like the divs above: `&lt;/div&gt;`

**Review by systimotic — CHANGES_REQUESTED** [review]
This is really good work! Thank you so much for updating the PR

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json (hunk 1 above)
Comment (szib) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Thanks for your help @systimotic You're absolutely right, I'll fix it soon and insert your superuser link as well.

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 2]
```diff
@@ -542,24 +542,17 @@
...
+        "First, the <code>${variable}</code> syntax. It's a template literal. Basically, you won't have to use concatenation with + anymore. Just drop the variable in your string, wrapped with ${ and }.",
         "Second, we used backticks, not quotes, to wrap around the string. Notice that the string is multi-line.",
         "In ES6, back-ticks give you more robust string creation ability.",
         "Instructions",
         "Use proper template literal syntax with backticks.",
-        "The expected output is each entry of result object's failure array, wrapped inside an <li> element, with class attribute text-warning.",
-        "If you have trouble finding backticks, it's the ` key, to the left of your 1; and has ~ on it."
+        "The expected output is each entry of result object's failure array, wrapped inside an &lt;li&gt; element, with class attribute text-warning.",
+        "Backtick key is usually located just above [TAB], to the left of your 1. If you have trouble finding it, read <a href=\"http://superuser.com/a/254077\" target=\"_blank\">this article</a>. "
```
Comment (systimotic) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]:
```diff
- Backtick key is usually located just above [TAB], to the left of your 1.
+ The backtick key is usually located right above \"tab\", to the left of the \"1\" key.
```

Comment (systimotic) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: There's a trailing space behind the last period. Sorry, I'm really nitpicking here 😅

Comment (systimotic) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Looks like superuser also serves over HTTPS, so let's link to that instead 👍 https://superuser.com/a/254077

**Review by systimotic — CHANGES_REQUESTED** [review]
This challenge will be much improved by your changes, thank you so much @szib!
I'm sorry, but I'm going to keep annoying you for a little bit longer 😅 
Thank you for staying on top of things. It's really helpful when people stay around and make the needed changes.

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json (hunk 2 above)
Comment (szib) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Thanks. 👍  I have fixed them now.

**Comment by raisedadead** [comment]
@systimotic I believe this is ready, except for the merge conflicts, which we can resolve. Should I do that and merge, after a sanity check?

**Comment by dhcodes** [comment]
@raisedadead yes, sanity check and merge after resolving conflicts if good to go.

**Comment by raisedadead** [comment]
@robbawebba @szib Can you guys confirm if the changes in this PR are valid. I was checking the merge conflict, and am unsure if the last change made to this file by @robbawebba has already taken care of what this PR was aiming.

Sorry about the delay in the QA.

**Comment by szib** [comment]
@raisedadead It seems to me @robbawebba rewrote this challenge since my PR. I suppose there's no need to merge this PR.

**Comment by raisedadead** [comment]
@szib thanks for the update and we value your contribution, please keep them coming. :smile:

**Comment by systimotic** [comment]
I apologize for disappearing @szib! I was busy with exams and haven't been around, but I'm back now. I look forward to seeing your contributions again!

**Pull request closed without merging**